frontend/util/datetime.js

```diff
@@ -28,52 +40,52 @@ function getUTCDate(dateString = Date.now()) {
  * @param  {String | Date} startTime  format "YYYY-DD-DDT00:00:00.000Z"
  * @return {String}                   human readable duration
  */
-export const duration = (startTime, endTime) => {
+export const duration = compat((startTime, endTime) => {
   if (!startTime) {
     return NO_TIME;
   }
 
   const baseTime = endTime || new Date();
-  return distanceInWords(baseTime, startTime);
-};
+  return formatDistance(startTime, baseTime);
+});
 
 
 /**
  * Return a human-readable time since the supplied date, like '5 hours ago'
  * @param  {String | Date} date format "YYYY-DD-DDT00:00:00.000Z"
  * @return {String}
  */
-export const timeFrom = (date) => {
+export const timeFrom = compat((date) => {
   if (!date) {
     return NO_TIME;
   }
 
-  return distanceInWordsStrict(new Date(), date, { addSuffix: true });
-};
+  return formatDistanceStrict(date, new Date(), { addSuffix: true, roundingMethod: 'floor' });
+});
 
 /**
  * Return a human-readable day, months and year (i.e. December 25th 2020 5:22 PM)
  * @param  {String | Date} date format "YYYY-DD-DDT00:00:00.000Z"
  * @return {String}
  */
-export const dateAndTime = (date) => {
+export const dateAndTime = compat((date) => {
   if (!date) {
     return NO_TIME;
   }
 
-  return format(date, 'MMMM Do YYYY, h:mm:ss a');
-};
+  return format(date, 'MMMM do yyyy, h:mm:ss aaaa');
+});
 
 /**
  * Return a formatted timestamp in UTC (i.e. '2018-02-11 04:22:33 ')
  * @param  {String | Date} date format "YYYY-DD-DDT00:00:00.000Z"
  * @return {String}
  */
-export const timestampUTC = (date) => {
+export const timestampUTC = compat((date) => {
   if (!date) {
     return NO_TIME;
   }
 
-  const utcTimestamp = format(getUTCDate(date), 'YYYY-MM-DD HH:mm:ss');
+  const utcTimestamp = format(getUTCDate(date), 'yyyy-MM-dd HH:mm:ss');
```

Comment:
format tokens changed https://blog.date-fns.org/post/unicode-tokens-in-date-fns-v2-sreatyki91jg/